apps/teams-test-app/src/components/AuthenticationAPIs.tsx

```diff
@@ -128,6 +128,38 @@ const Authenticate = (): React.ReactElement =>
     },
   });
 
+const externalAuthParams = {
```

Comment:
Why do we need these to be hardcoded here? They seem relevant to the E2E scenarios tests you're writing, and should be typed into the input box by the test instead of the test app having them hardcoded.

Reply:
updated